syntax = "proto3";

package ai.v1;

import "ai/ai_conversation.proto";

// AI Conversation Service
service AIConversationService {
  // Sends a message to the AI and receives a streaming response
  rpc SendMessage(SendMessageRequest) returns (stream SendMessageResponse) {};
  
}